configure.ac

```diff
@@ -1038,6 +1040,7 @@ examples/platforms/cc1352/Makefile
 examples/platforms/cc2538/Makefile
 examples/platforms/cc2650/Makefile
 examples/platforms/cc2652/Makefile
+examples/platforms/efr32mg1/Makefile
```

Comment:
Actually, after thinking about it a little more and seeing this comment 
```
MG1 cannot fit FTD builds with certain options enabled (like diagnostics). In testing, I simply removed FTD from build options. We may want to add this to README
```
I think you're right. Let's remove the things that can't build and run due to size issues.

Reply:
I had issues building throughout testing this, but I didn't attempt to add sleepy demo back at the end and see if it would build. I was only interested in RCP builds, frankly, but defer to you and Silabs team on preference.

Reply:
Yes, let's leave the sleepy demo support out.